src/components/sbb-expansion-panel/sbb-expansion-panel.stories.tsx

```diff
@@ -0,0 +1,158 @@
+/** @jsx h */
```

Comment:
when using borderless and white, it would be nice to have a darker background so it stays visible

Reply:
Fixed in commit a9406aff